modules/logging-release-notes-5-8-3.adoc

```diff
@@ -0,0 +1,20 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-3_{context}"]
+= Logging 5.8.3
+This release includes link:https://access.redhat.com/errata/RHBA-2024:0693[Logging Bug Fix 5.8.3]
+
+[id="logging-release-notes-5-8-3-bug-fixes"]
+== Bug fixes
+(None.)
```

Comment:
You are looking only on the RHEL-8 errata here. This this the reason there are no fixes. In fact we need always to point to both errata in 5.8 (RHEL-8 and RHEL-9). Latter is here and has issues to be attached: https://errata.engineering.redhat.com/advisory/126476